config/config.go

```diff
@@ -679,6 +680,7 @@ type EC2SDConfig struct {
 	SecretKey       string   `yaml:"secret_key,omitempty"`
 	RefreshInterval Duration `yaml:"refresh_interval,omitempty"`
 	Port            int      `yaml:"port"`
+	SubnetSeparator string   `yaml:"subnet_separator,omitempty"`
```

Comment:
Right. The subnet id can be only in the form "subnet-[a-z0-9]+", so it should be safe to leave comma as a fixed separator. Will wait a few hours in case someone else has some other comments, than I will push a new change removing the SubnetSeparator configuration parameter.